views/partials/layout.njk

```diff
@@ -5,7 +5,12 @@
 {% block head %}
 
 <!--[if !IE 8]><!-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js" type="text/javascript"></script>
+<script src="http://code.jquery.com/jquery-migrate-1.2.1.js"></script>
+<link href = "https://code.jquery.com/ui/1.12.1/themes/ui-lightness/jquery-ui.css" rel = "stylesheet">
 <link href="/static/stylesheets/application.css" rel="stylesheet" />
+<script src="/static/js/main.js"></script>
```

Comment:
shouldn't scripts go at the bottom of the page - not sure if this is a nunjucks thing though...

Reply:
No, this can be moved.